src/spend.rs

```diff
@@ -462,11 +462,37 @@ pub enum SpendTxFees {
     Rbf(u64, u64),
 }
 
+pub enum CreateSpendWarning {
+    /// A possible change output has not been included.
+    ChangeNotIncluded(u64),
+    /// Excess is insufficient to pay for a change output.
+    ExcessTooSmallForChange(u64),
```

Comment:
As a consequence of my previous comment, this would be a single variant. I think it would be a good thing, it's really unclear what the difference between the two is here.

Reply:
I've now combined them into a single `FeeOverpayment`.